trantor/utils/SerialTaskQueue.h

```diff
@@ -61,7 +62,7 @@ class SerialTaskQueue : public TaskQueue
      *
      * @param name
      */
-    explicit SerialTaskQueue(const std::string &name = std::string());
+    explicit SerialTaskQueue(const std::string &name);
```

Comment:
Why remove the default value here?

Reply:
When you cann `SerialTaskQueue();` there is an ambiguity between these two constructors:

```
    SerialTaskQueue() = delete;
``` 
and 
```
    explicit SerialTaskQueue(const std::string &name = std::string());
```

Both constructors can be called with no parameters.

Reply:
I see, you are right. Thanks.